assets/dev/js/editor/views/add-section/base.js

```diff
@@ -119,10 +119,22 @@ class AddSectionBase extends Marionette.ItemView {
 			placeholder: false,
 			currentElementClass: 'elementor-html5dnd-current-element',
 			hasDraggingOnChildClass: 'elementor-dragging-on-child',
-			onDropping: this.onDropping.bind( this ),
+			// Merge different options if provided by child elements
+			...this.getDroppableOptions?.(),
```

Comment:
Why do you need this `?.`, Isn't it always defined?

Reply:
Previously this method was only existed in the `inline.js` file. Now when it's implemented here too, you're right, `?.` has no importance.
Fixed.